addons/sale_management/views/sale_order_views.xml

```diff
@@ -102,9 +102,11 @@
 
             <field name="partner_shipping_id" position="after">
                 <field name="sale_order_template_id"
-                    options="{'no_create': True}"
-                    groups="sale_management.group_sale_order_template"
-                 readonly="state in ['cancel', 'sale']"/>
+                       options="{'no_create': True}"
+                       groups="sale_management.group_sale_order_template"
+                       readonly="state in ['cancel', 'sale']"
+                       invisible="state in ['cancel', 'sale', 'done']"
```

Comment:
Does it make sense for this field to be both invisible and readonly when `state in ['cancel', 'sale']`? 
I also don't think it is a change we want in this task.

Reply:
:+1: This was originally a change requested by dala but he told me you convinced him otherwise, will remove the `invisible`